integration_tests/tests/in_cluster.rs

```diff
@@ -118,68 +123,99 @@ impl InClusterJanusPair {
+            .create_shared_aggregator(NewSharedAggregator {
                 name: "leader".to_string(),
-                api_url: Self::in_cluster_aggregator_api_url(&leader_namespace).to_string(),
+                api_url: Self::in_cluster_aggregator_api_url(&leader_namespace),
                 bearer_token: leader_aggregator_api_auth_token,
+                is_first_party: true,
             })
-            .await;
+            .await
+            .unwrap();
 
         let paired_helper_aggregator = divviup_api
-            .pair_aggregator(
-                &account,
-                &NewAggregatorRequest {
+            .create_aggregator(
+                account.id,
+                NewAggregator {
                     name: "helper".to_string(),
-                    api_url: Self::in_cluster_aggregator_api_url(&helper_namespace).to_string(),
+                    api_url: Self::in_cluster_aggregator_api_url(&helper_namespace),
                     bearer_token: helper_aggregator_api_auth_token,
                 },
             )
-            .await;
+            .await
+            .unwrap();
 
+        let hpke_config = task.collector_hpke_config().unwrap();
         let collector_hpke_config = divviup_api
             .create_hpke_config(
-                &account,
-                &NewHpkeConfigRequest {
-                    name: "Integration test key".to_string(),
-                    contents: STANDARD.encode(task.collector_hpke_config().unwrap().get_encoded()),
-                },
+                account.id,
+                HpkeConfigContents::new(
+                    (hpke_config.id() as u8).into(),
+                    (hpke_config.kem_id() as u16).try_into().unwrap(),
+                    (hpke_config.kdf_id() as u16).try_into().unwrap(),
+                    (hpke_config.aead_id() as u16).try_into().unwrap(),
+                    hpke_config.public_key().as_ref().to_vec().into(),
+                ),
+                Some("Integration test key"),
             )
-            .await;
+            .await
+            .unwrap();
 
-        let provision_task_request = NewTaskRequest {
+        let provision_task_request = NewTask {
             name: "Integration test task".to_string(),
             leader_aggregator_id: paired_leader_aggregator.id,
             helper_aggregator_id: paired_helper_aggregator.id,
-            vdaf: task.vdaf().try_into().unwrap(),
+            vdaf: match task.vdaf().to_owned() {
+                VdafInstance::Prio3Count => Vdaf::Count,
+                VdafInstance::Prio3Sum { bits } => Vdaf::Sum {
+                    bits: bits.try_into().unwrap(),
+                },
+                VdafInstance::Prio3SumVec { bits, length } => Vdaf::SumVec {
+                    bits: bits.try_into().unwrap(),
+                    length: length.try_into().unwrap(),
+                },
+                VdafInstance::Prio3Histogram { length } => Vdaf::Histogram {
+                    buckets: (0..*length - 1)
```

Comment:
`length` is a `usize` and not a reference in this case
```suggestion
                    buckets: (0..length - 1)
```